wordfreq/transliterate.py

```diff
@@ -0,0 +1,70 @@
+# This table comes from https://github.com/opendatakosovo/cyrillic-transliteration/blob/master/cyrtranslit/mapping.py,
+# from the 'cyrtranslit' module, which is too flaky for us to just import.
```

Comment:
Given that this is for public release, I wonder if there's a less judgmental way to put this.